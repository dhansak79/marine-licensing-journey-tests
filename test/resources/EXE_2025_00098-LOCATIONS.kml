<?xml version="1.0" encoding="UTF-8"?>
<kml xmlns="http://www.opengis.net/kml/2.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/kml/2.2 http://schemas.opengis.net/kml/2.2.0/ogckml22.xsd">
<Folder>
<name>EXE/2025/00098</name>
<Document id="SL914071">
<name>mmo_advice_sites</name>
<description>MMO Advice Sites</description>
<Style id="s_advice_main_line_point">
<LineStyle><color>FF000000</color></LineStyle>
<PolyStyle><color>FFFFFFFF</color></PolyStyle>
</Style>
<Placemark id="SID12796347">
<name>Kentish Flats and Kentish Flats Extension</name>
<description>Kentish Flats and Kentish Flats Extension</description>
<styleUrl>#s_advice_main_line_point</styleUrl>
<Polygon>
<outerBoundaryIs>
<LinearRing><coordinates>1.02333333333333333333333333333333333334,51.46906666666666666666666666666666666667,0 1.06491666666666666666666666666666666667,51.43608333333333333333333333333333333333,0 1.15086666666666666666666666666666666667,51.44465,0 1.10945,51.47773333333333333333333333333333333334,0 1.02333333333333333333333333333333333334,51.46906666666666666666666666666666666667,0 </coordinates></LinearRing>
</outerBoundaryIs>
</Polygon>
</Placemark>
<Placemark id="SID12796357">
<name>Thanet Offshore Wind Farm</name>
<description>Thanet Offshore Wind Farm</description>
<styleUrl>#s_advice_main_line_point</styleUrl>
<Polygon>
<outerBoundaryIs>
<LinearRing><coordinates>1.56861666666666666666666666666666666667,51.44065,0 1.62476666666666666666666666666666666667,51.40178333333333333333333333333333333333,0 1.67171666666666666666666666666666666667,51.3993,0 1.6872,51.4033,0 1.68791666666666666666666666666666666666,51.42575,0 1.6379,51.46043333333333333333333333333333333333,0 1.62093333333333333333333333333333333334,51.46131666666666666666666666666666666667,0 1.60503333333333333333333333333333333333,51.46215,0 1.59275,51.45896666666666666666666666666666666667,0 1.57651666666666666666666666666666666667,51.4498,0 1.56861666666666666666666666666666666667,51.44065,0 </coordinates></LinearRing>
</outerBoundaryIs>
</Polygon>
</Placemark>
</Document>
</Folder>
</kml>
