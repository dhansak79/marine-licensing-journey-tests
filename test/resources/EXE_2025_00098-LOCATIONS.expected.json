{
  "extractedSites": [
    {
      "siteName": "Kentish Flats and Kentish Flats Extension",
      "extractedCoordinates": [
        [1.0233333333333334, 51.46906666666667, 0],
        [1.0649166666666667, 51.436083333333336, 0],
        [1.1508666666666667, 51.44465, 0],
        [1.10945, 51.47773333333333, 0],
        [1.0233333333333334, 51.46906666666667, 0]
      ],
      "geometryType": "Polygon"
    },
    {
      "siteName": "Thanet Offshore Wind Farm",
      "extractedCoordinates": [
        [1.5686166666666668, 51.44065, 0],
        [1.6247666666666667, 51.401783333333334, 0],
        [1.6717166666666667, 51.3993, 0],
        [1.6872, 51.4033, 0],
        [1.6879166666666667, 51.42575, 0],
        [1.6379, 51.460433333333334, 0],
        [1.6209333333333333, 51.46131666666667, 0],
        [1.6050333333333333, 51.46215, 0],
        [1.59275, 51.45896666666667, 0],
        [1.5765166666666666, 51.4498, 0],
        [1.5686166666666668, 51.44065, 0]
      ],
      "geometryType": "Polygon"
    }
  ],
  "description": "Multi-site KML file with 2 offshore wind farm sites in Thames Estuary area"
}
